tests/integration/web/maintenance/utils_test.py

```diff
@@ -0,0 +1,29 @@
+from nav.models.manage import Netbox, Room
+from nav.web.maintenance.utils import prefetch_and_group_components
+
+
+class TestPrefetchAndGroupComponents:
+    def test_groups_netboxes_by_room(self, db, client):
```

Comment:
You'll find I like to nitpick test names quite a lot ;-)

```suggestion
    def test_should_group_netboxes_by_room(self, db, client):
```